performance_curves/performance_curve.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from typing import Callable, Optional
+
+from performance_curves.utils import synchronize_sort
+
+
+class PerformanceCurve:
+    """Stores values of the x- and y-axis of the performance curve and allows plotting"""
+    def __init__(self, cutoff, metric_value):
+        self.cutoff = cutoff
+        self.metric_value = metric_value
+
+    def plot(self):
+        fig, ax = plt.subplots()
+        ax.plot(self.cutoff, self.metric_value)
+        ax.set_xlim([1., 0.])
+        ax.set_ylim([0., 1.10])
+        ax.set_xticks([i for i in reversed(np.arange(0, 1, 0.1))])
+        ax.set_yticks([i for i in np.arange(0, 1.01, 0.1)])
+        ax.set_xlabel('Probability Estimates in Descending Order')
+        ax.set_ylabel('Metric Value')
+        ax.set_title('Performance Curve')
+        plt.show()
+
+
+def performance_curve(
+    y_true: np.ndarray,
+    y_score: np.ndarray,
+    metric: Callable[[np.ndarray, np.ndarray], float],
+    num_bins: Optional[int] = None
+) -> PerformanceCurve:
+    """
```

Comment:
Docstrings should have a one-liner description.

Reply:
Added!